### DISCUSSION TITLE
Question about SEB Runtime Logs Transmission

### DISCUSSION THREAD

**Discussion by Lexxn0x3** [SafeExamBrowser/seb-server#131] [Q&A] [answered]
Hello SEB Development Team,

We are working on a student project to analyze SEB log files to detect cheating attempts. We understand that SEB stores client and runtime logs, and we can access parts of the client log on the SEB server. However, we are particularly interested in the runtime logs, which include information such as system information, integrity checks, number of displays, etc..

Our question is:
- Are the runtime logs transmitted to the SEB server?
- If yes, how can we access these runtime logs on the SEB server?

Thank you for your assistance.

Best regards,

Robin
Technische Hochschule Nürnberg Georg Simon Ohm
CYBRAIL

**Comment by anhefti** [comment]
Hello Robin,

Thank you for your interest in SEB and SEB Server.
If you prepare an Exam with SEB Server, you can configure the logging of SEB Clients and what logs they send to SEB Server within the Exam Configuration of the Exam. In The "Security" section of the SEB Settings on your Exam Configuration, you will see the Logging settings:

![image](https://github.com/SafeExamBrowser/seb-server/assets/39078777/8055faba-0153-4d5c-95ab-6565826bde43)
 
Default settings is on "Warning" level. So you will only have "Warning" and "Error" logs on SEB Server (some "Info" logs might be from SEB SEB Server signalling for indicators or SEB Server logs for the clients)

Beside of that you can activate "Show log button on taskbar (Win)" for Windows Clients. So you will have a Button inside your SEB to open a window with the SEB Client logs.

On SEB Server you can then access the SEB Client logs within the "Monitoring" section. If your Exam is running, go to Running Exams, if it has been finished, go to Finished Exams and double-click on the SEB Client connection of your interest to see the logs of a particular SEB Client or for finished Exams you also can export the SEB Client logs of the whole exam by using "Export SEB Logs" within CSV format.
Finally, you can search all SEB Logs within the "SEB Client Logs" section under "Monitoring" but this might be slow when there are millions of logs.

For more precise answer on about what Logs are sent to SEB Server (depending on Log Level), let me also ask our SEB Client developers: @dbuechel @danschlet 

Best regards
Andreas

**Reply by dbuechel** [reply, marked as answer]
In SEB for Windows, only the logs of the client application component (running the main UI and browser of SEB) are sent to the server according to the defined log severity.

**Reply by Lexxn0x3** [reply]
Thank you very much!
